content/en/blog/2019/app-identity-and-access-adapter/index.md

```diff
@@ -0,0 +1,167 @@
+---
+title: App Identity and Access Adapter
+subtitle: Using Istio to Secure Multicloud Kubernetes Applications with Zero Code Changes
+description: Using Istio to Secure Multicloud Kubernetes Applications with Zero Code Changes.
+publishdate: 2019-09-01
+attribution: Anton Aleksandrov (IBM)
+keywords: [security,oidc,jwt,policies]
+---
+
+Whether your computing environment uses multiple cloud providers, a single cluster, a combination of on- and off-premise solutions, or multiple containers in one cloud, having a centralized identity management can help you to preserve existing infrastructure and avoid vendor lock-in.
```

Comment:
I think the most compelling/attractive part of this is "zero code changes or re-deployments", you might lead with that instead and work these other benefits into later in the content.